core/ui/Actions/new-image.tid

```diff
@@ -6,5 +6,5 @@ description: create a new image tiddler
 image/$(imageType)$
 \end
 <$vars imageType={{$:/config/NewImageType}}>
-<$action-sendmessage $message="tm-new-tiddler" type=<<get-type>>/>
+<$action-sendmessage $message="tm-new-tiddler" type=<<get-type>> tags={{$:/language/DefaultNewTiddlerTags!!tags}}/>
```

Comment:
Hi @BurningTreeC I don't think that this tiddler should be in the `$:/language/` namespace because it's not part of the set of translatable strings. Instead it should be `$:/config/NewTiddler/tags`.

Reply:
ok @Jermolene

Reply:
now it's using the tiddler `$:/config/NewTiddler/Tags` with Tags uppercase like `$:/config/NewJournal/Tags`